src/plugins/visualizations/server/migrations/visualization_saved_object_migrations.ts

```diff
@@ -1102,4 +1136,5 @@ export const visualizationSavedObjectTypeMigrations = {
     migrateTagCloud,
     replaceIndexPatternReference
   ),
+  '7.15.0': flow(removeMarkdownLessFromTSVB),
```

Comment:
We cannot skip version. Maybe someone can provide idea how we should add migration script for 8.0.0?